### PR TITLE
Second update README.md

### PR DESCRIPTION
**Pull request by MHolub2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Hash function by Adler and Fletcher algorithm
```
Comment (MaksGolub) [inline README.md]: Good Job!

Comment (MaksGolub) [inline README.md]: Please, edit from "Hash" to "Checksum"

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# Hash function by Adler and Fletcher algorithm 
+# Checksum function by Adler and Fletcher algorithm
```
Comment (MaksGolub) [inline README.md]: Good Job!

**Comment by MHolub2** [comment]
Second update pull request

**Pull request closed without merging**